SPECS/swtpm/swtpm.spec

```diff
@@ -0,0 +1,365 @@
+%bcond_without gnutls
+
+%global gitdate     20230815
+%global gitcommit   d2849a9f5ced70438d67036693438344b47b4161
+%global gitshortcommit  %(c=%{gitcommit}; echo ${c:0:7})
+
+# Macros needed by SELinux
+%global selinuxtype targeted
+%global moduletype  contrib
+%global modulename  swtpm
+
+Summary: TPM Emulator
+Name:           swtpm
+Version:        0.8.1
+Release:        4%{?dist}
+License:        BSD-3-Clause
+Url:            http://github.com/stefanberger/swtpm
+Source0:        %{url}/archive/%{gitcommit}/%{name}-%{gitshortcommit}.tar.gz
+
+BuildRequires:  make
+BuildRequires:  git-core
+BuildRequires:  automake
+BuildRequires:  autoconf
+BuildRequires:  libtool
+BuildRequires:  libtpms-devel >= 0.6.0
+BuildRequires:  expect
+BuildRequires:  net-tools
+BuildRequires:  openssl-devel
+BuildRequires:  socat
+%if %{undefined rhel}
```

Comment:
I think we want to either remove the if rhel sections entirely, or change it to if azure (so this would be %if {undefined azure} and we'll need to define azure elsewhere. For now probably it's best to just remove this section. Unless, of course, we *do* want to build with trousers, but that's basically dead/replaced so i don't think we really want to.

If you check rawhide, they've already completely removed this trousers dep.